src/fu-util.c

```diff
@@ -2304,7 +2304,7 @@ fu_util_maybe_send_reports(FuUtilPrivate *priv, const gchar *remote_id, GError *
 }
 
 static gboolean
-fu_util_update_all(FuUtilPrivate *priv, GError **error)
+fu_util_update_all(FuUtilPrivate *priv, gchar **values, GError **error)
```

Comment:
since you essentially combined `fu_util_update_all` and `fu_util_update_by_id` into a single function suggest this function should just move into `fu_util_update`

Reply:
It looks like `fu_util_update` already in use, how about use `fu_util_update_runner` instead?

Reply:
Yeah but this function is only called by the old fu_util_update right? So just put it inline in that function perhaps.